ONSdigital/sdx-compose
# Hh xml

**Changes**

sdx-transform-testform - Template and endpoint added for HH questionnaires
sdx-console - HH survey added
sdx-downstream - able to triage and process HH questionnaires

**How to test**

 Checkout branch 'hh-xml' for all submodules named above. In console select survey '0.hh2016.json' and'Submit'. View the message via the 'sdx-testform' queue in RabbitMQ Management, and assert that it is returning the correct format - see 
[HHUnitData.txt](https://github.com/ONSdigital/sdx-transform-testform/files/396154/HHUnitData.txt)

**Who can test**

Anyone but @jjeane
